universal-application-tool-0.0.1/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java

```diff
@@ -102,4 +103,32 @@ public boolean preferredLanguageSupported() {
 
     return blockListBuilder.build();
   }
+
+  @Override
+  public ImmutableList<AnswerData> getSummaryData() {
+    ImmutableList.Builder<AnswerData> builder = new ImmutableList.Builder<AnswerData>();
+    ImmutableList<Block> blocks = getAllBlocks();
+    for (Block block : blocks) {
+      String blockId = block.getId();
+      for (ApplicantQuestion question : block.getQuestions()) {
+        String questionText = question.getQuestionText();
```

Comment:
(blocking) `ApplicantQuestion#getQuestionText` uses `applicantData#preferredLocale()`, which is great for an applicant facing review page, but we'll want the admin preferred locale for admin facing things (e.g. the admin application views, or exports).

One option is to add a `boolean forAdmin` argument to this method, and add `getQuestionTextForAdmin` and `getQuestionHelpTextForAdmin` methods to `ApplicantQuestion` and switch between using those.

Reply:
A TODO would suffice since this is a bit outside the scope of your PR.